frontend/packages/dev-console/integration-tests/features/topology/workload-Groupings-ODC-2922.feature

```diff
@@ -0,0 +1,81 @@
+Feature: Workload Groupings in Topology 
+    User will be able to expand and collapse all groups on Topology graph and list view
+
+
+Background:
+    Given user is at developer perspecitve
+
+
+@regression, @smoke
+Scenario: Default state of Display dropdown
+    Given user is at the Topology page
+    When user clicks on the Display dropdown
+    Then user will see the Show Groups is checked
+    And user will see the Expand is checks
+    And user will see the Knative Services checkbox checked
+    And user will see the Helm Releses checkbox checked
+    And user will see the Operator Groups checkbox checked
+
+
+@regression, @smoke
+Scenario: Uncheck the Show Groups
+    Given user is at the Topology page
+    When user clicks on the Display dropdown
+    And user unchecks the Show Groups
+    Then user will see that the Expand and it's children are disabled
```

Comment:
User will see that group no longer appear in the view, only workloads (both standalone and those belonging to groupings)

Reply:
Updated the statement